packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json

```
@@ -13,5 +13,8 @@
   "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents!",
   "user_account_identity_documents_submit_success_message_description": "Nous avons bien reçu l’ensemble de vos documents. Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d’informations complémentaires.",
   "user_account_identity_documents_submit_error_message_description": "Vos documents n’ont pas été correctement envoyés. Veuillez refaire l’upload et l’envoi de vos documents",
-  "user_account_identity_documents_go_back_to_dashboard": "Voir mon tableau de bord"
+  "user_account_identity_documents_go_back_to_dashboard": "Voir mon tableau de bord",
+  "user_account_identity_documents_legal_info1": "OVH S.A.S est le responsable du traitement de vos données personnelles. Les données collectées via ce formulaire sont traitées afin de se conformer aux <a href=\"{{ link1 }}\" target=\"_blank\">Directives de cybersécurité du 28.04.2022</a> publiées par le CERT-IN en vertu des dispositions de la sous-section (6) de <a href=\"{{ link2 }}\" target=\"_blank\">l'article 70B</a> de la Loi de 2000 ",
```

Related to law documentation we don't need the separation point.
To be confirmed with PM, PO.
https://www.economie.gouv.fr/entreprises/societe-actions-simplifiee-SAS#:~:text=Une%20SAS%20peut%20%C3%AAtre%20constitu%C3%A9e,actionnaires%20est%20limit%C3%A9e%20aux%20apports.

```suggestion
  "user_account_identity_documents_legal_info1": "OVH SAS est le responsable du traitement de vos données personnelles. Les données collectées via ce formulaire sont traitées afin de se conformer aux <a href=\"{{ link1 }}\" target=\"_blank\">Directives de cybersécurité du 28.04.2022</a> publiées par le CERT-IN en vertu des dispositions de la sous-section (6) de <a href=\"{{ link2 }}\" target=\"_blank\">l'article 70B</a> de la Loi de 2000 ",
```